examples/firebase-functions/src/index.ts

```diff
@@ -0,0 +1,30 @@
+import * as functions from 'firebase-functions'
+import { App } from '@tinyhttp/app'
+const tinyhttp = new App()
+
+/** Your application code */
+tinyhttp.use('/', (_, res) => {
+  res.send('<h1>Hello World</h1>')
+})
+
+// Create and Deploy Your First Cloud Functions
+// https://firebase.google.com/docs/functions/write-firebase-functions
+export const app = functions.https.onRequest((request, response) => {
+  makeReadonlySettable(request)
+  // @ts-ignore
```

Comment:
you could do `request: Request` or `request as Request`.

there's no reason to use `@ts-ignore`